multilinear_extensions/src/mle.rs

```diff
@@ -123,14 +123,13 @@ impl<F: Field, E: ExtensionField<BaseField = F>> IntoMLEs<DenseMultilinearExtens
 }
 
 #[derive(Clone, PartialEq, Eq, Hash, Default, Debug, Serialize, Deserialize)]
-#[serde(untagged)]
 #[serde(bound(
     serialize = "E::BaseField: Serialize",
     deserialize = "E::BaseField: DeserializeOwned"
 ))]
 /// Differentiate inner vector on base/extension field.
 pub enum FieldType<E: ExtensionField> {
-    Base(#[serde(skip)] Vec<E::BaseField>),
+    Base(Vec<E::BaseField>),
```

Comment:
understood this is to serve trivial proof for small polynomial 👍

Reply:
yes